external/storm-hdfs/src/test/java/org/apache/storm/hdfs/spout/TestHdfsSpout.java

```diff
@@ -704,12 +703,12 @@ public MockCollector() {
 
     @Override
     public void emitDirect(int arg0, String arg1, List<Object> arg2, Object arg3) {
-      throw new NotImplementedException();
+      throw new RuntimeException("NOT Implemented");
```

Comment:
I prefer using UnsupportedOperationException in this case but since  it's just a test code I'm OK.